docs/site/sitemap.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
+  <url>
+    <loc>https://tinkerpop.apache.org/</loc>
+    <lastmod>2025-05-07</lastmod>
```

Comment:
Will we need to manually maintain these `<lastmod>` tags as we update the site in the future?

Reply:
Yes (perhaps we can automate later).